lib/babel-config.js

```diff
@@ -13,19 +13,33 @@ const syntaxAsyncGeneratorsPath = require.resolve('@babel/plugin-syntax-async-ge
 const syntaxObjectRestSpreadPath = require.resolve('@babel/plugin-syntax-object-rest-spread');
 const transformTestFilesPath = require.resolve('@ava/babel-preset-transform-test-files');
 
+function validateKeys(conf) {
+	const keys = Object.keys(conf);
+	for (let i = 0; i < keys.length; i++) {
+		if (keys[i] !== 'testOptions' && keys[i] !== 'extensions') {
+			return false;
+		}
+	}
+	return true;
+}
+
 function validate(conf) {
 	if (conf === false) {
 		return null;
 	}
 
 	if (conf === undefined) {
-		return {testOptions: {}};
+		return {extensions: ['js'], testOptions: {}};
 	}
 
-	if (!conf || typeof conf !== 'object' || !conf.testOptions || typeof conf.testOptions !== 'object' || Array.isArray(conf.testOptions) || Object.keys(conf).length > 1) {
+	if (!conf || typeof conf !== 'object' || !conf.testOptions || typeof conf.testOptions !== 'object' || Array.isArray(conf.testOptions) || !validateKeys(conf)) {
```

Comment:
Instead of multiple `if` conditions we could use `invalid` instead then split them up by doing:
```js
let invalid = false;

invalid = !conf || typeof conf !== 'object' || ...;
invalid = ...

if (invalid) {
    throw ...
}
```
This should be a bit shorter than a bunch of if statements.